src/service/variable-source.js

```diff
@@ -44,15 +80,40 @@ const LOAD_EVENT_END = 'loadEventEnd';
  * @return {!Promise<ResolverReturnDef>}
  */
 export function getTimingDataAsync(win, startEvent, endEvent) {
-  let readyPromise = loadPromise(win);
-  if (startEvent === LOAD_EVENT_END || endEvent === LOAD_EVENT_END) {
+  // Fallback to load event if we don't know what to wait for
+  const startWaitForEvent =
+    NAV_TIMING_WAITFOR_EVENTS[startEvent] || WAITFOR_EVENTS.LOAD;
+  const endWaitForEvent = endEvent
+    ? NAV_TIMING_WAITFOR_EVENTS[endEvent] || WAITFOR_EVENTS.LOAD
+    : null;
+
+  let waitForEvent;
+  if (!endWaitForEvent) {
+    waitForEvent = startWaitForEvent;
+  } else {
+    waitForEvent = Math.max(startWaitForEvent, endWaitForEvent);
```

Comment:
Shouldn't end always be larger than start?

Reply:
Not always, The value could come from publishers. For example `${navTiming(end, start)}`. 
I don't expect anyone to actually use that, but want to keep the `Math.max` to make it always correct.